app/services/container_topology_service.rb

```diff
@@ -79,7 +79,7 @@ def entity_status(entity)
     elsif entity.kind_of?(ContainerReplicator)
       status = (entity.current_replicas == entity.replicas) ? 'OK' : 'Warning'
     elsif entity.kind_of?(ManageIQ::Providers::ContainerManager)
-      status = entity.authentications.empty? ? 'Unknown' : entity.authentications.first.status.capitalize
+      status = entity.authentications.empty? ? 'Unknown' : entity.authentications.find_by_authtype('bearer').status.capitalize
```

Comment:
The lines
```
status = entity.authentication_status
status = 'Unknown' if status == 'None'
``` 
can replace the line:
`entity.authentications.empty? ? 'Unknown' : entity.authentications.find_by_authtype('bearer').status.capitalize`